pints/_mcmc/__init__.py

```diff
@@ -651,6 +728,44 @@ def set_max_iterations(self, iterations=10000):
                     'Maximum number of iterations cannot be negative.')
         self._max_iterations = iterations
 
+    def set_output_files(self, chain_file, evaluation_file):
```

Comment:
Might be nicer to split this into two functions, `set_output_chain_file` and `set_output_evaluation_file`.

Reason being is that the name `set_output_files` is not very descriptive, does it refer to the logging perhaps?, and if you come across this in your code

```python
sampler.set_output_files(filename, None)
```

you have no idea what you are outputing....

Reply:
Fair enough. I want to avoid confusion with the other _log_ methods though (which have nothing to do with this!)

Reply:
i was thinking that having the words `chain` and `evaluation` might further distinguish it from the logging functions...

Reply:
Ok, have updated them now!

Reply:
Not sure that `evaluation` is very clear, I'd be expecting something more like `target` or `target_value`?

Reply:
Well we don't call it target anywhere else, plus the target is surely the distribution itself, not a single point within it? Happy to change it to say `set_logpdf_filename(...)`